tests/template_literals/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,5 +1,69 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`css-prop.js 1`] = `
+function SomeComponent (props) {
+  // Create styles as if you're calling css and the class will be applied to the component
+  return (<div css={\`
+    color: blue;
+    font-size: 17 px;
+
+    &:hover {
+      color: green;
+    }
+
+    & .some-class {
+      font-size: 20px;
+    }
+  \`}>
+    This will be blue until hovered.
+    <div className="some-class">
+      This font size will be 20px
+    </div>
+  </div>)
+}
+
+const TestComponent = ({ children, ...props }) => (
+  <div css={\`color: white; background: black\`}>
+    {children}
+  </div>
+);
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+function SomeComponent(props) {
+  // Create styles as if you're calling css and the class will be applied to the component
+  return (
+    <div
+      css={\`
+        color: blue;
+        font-size: 17 px;
```

Comment:
I could swear this number + unit space was normalized (either always print or always omit)

Reply:
I don't think that's a safe normalisation to do.